smarts/core/renderer.py

```diff
@@ -58,6 +58,7 @@ def __new__(cls):
             # disable vsync otherwise we are limited to refresh-rate of screen
             loadPrcFileData("", "sync-video false")
             loadPrcFileData("", "model-path %s" % os.getcwd())
+            loadPrcFileData("", "model-cache-dir %s/.panda3d_cache" % os.getcwd())
```

Comment:
is this related to traffic data?

Reply:
Oops.  No, it's not.  I accidentally left that in after a test.  I'll take it back out.  Thanks for catching that!

Reply:
(It did speed up rendering though, so we might consider adding it at some point.)